website/docs/addon_slack.md

```diff
@@ -15,83 +15,130 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 
-This module allows configuring profiles(when to trigger, for which combination of task, host and family)
-and templates(could contain {} placeholder) to send notification to Slack channel(s)
-whenever configured asset type is published.
+This addon enables you to set up profiles that specify when to trigger notifications, and for which combination of task, host, and product types.
+It also supports customizable message templates that use placeholders (denoted by curly brackets `{}`) that are automatically populated at the time of publishing.
 
 
 ## App installation
 
-Slack application must be installed to company's Slack first. 
+Please ensure that the Slack application is installed on the company's Slack before proceeding.
```

Comment:
I got quite confused reading over this page because it leaves a bit in the middle which section applies to Slack and which applies to AYON.

I wonder if phrasing it like this helps clarify what part of the installation is "slack" related and when you're moving to AYON specifics.
```suggestion
Please ensure that the AYON Slack application is installed on the company's Slack using the `manifest.yml` before proceeding to AYON settings.
```